pkg/rbac/authenticationMiddleware.go

```diff
@@ -33,6 +34,7 @@ func AuthenticateUser(next http.Handler) http.Handler {
 			klog.V(4).Info("Request didn't have a valid authentication token.")
 			http.Error(w, "{\"message\":\"Request didn't have a valid authentication token.\"}",
 				http.StatusUnauthorized)
+			metric.AuthnFailed.WithLabelValues("ValidAuthntokenMissing").Inc()
```

Comment:
Is the http response code reported "for free" ?
I wonder if these are too similar to the response codes (http.StatusUnauthorized, http.StatusForbidden)

Reply:
I captured the http status code in the metrics now instead of plain text.